pivot.coffee

```diff
@@ -409,6 +410,13 @@ callWithJQuery ($) ->
                 len++
             return len
 
+        #cell click callback generator
+        makeClickCallback = (cb,cellValue,attributes,keys,data) ->
+            filters = {}
+            for own key,attr of attributes
+                filters[attr] = keys[key]
+            return (e) -> cb(e,cellValue,filters,data)
```

Comment:
not sure about this signature... What are we trying to enable within the callback? Mostly I think people want to be able to extract the matching records, but providing the data in this way doesn't seem like the easiest way to enable that.